deploy/README.md

```diff
@@ -1,57 +1,66 @@
 # Deployment Guide
 
-This page has instructions for collecting Kubernetes logs, metrics, and events; enriching them with deployment, pod, and service level metadata; and sending them to Sumo Logic.
+This page has instructions for collecting Kubernetes logs, metrics, and events; enriching them with deployment, pod, and service level metadata; and sending them to Sumo Logic. See our [documentation guide]([https://help.sumologic.com/Solutions/Kubernetes_Solution](https://help.sumologic.com/Solutions/Kubernetes_Solution)) for details on our Kubernetes Solution.
 
 **Compatibility**: Kubernetes version 1.11+
```

Comment:
Since this is encapsulated in the `minimum requirements`, let's get rid of this here